test/metric/processes.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package metric
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+)
+
+type ProcessesMetricValueFetcher struct {
+	baseMetricValueFetcher
+}
+
+var _ MetricValueFetcher = (*ProcessesMetricValueFetcher)(nil)
+
+func (f *ProcessesMetricValueFetcher) Fetch(namespace, metricName string, stat Statistics) (MetricValues, error) {
+	dims := f.getMetricSpecificDimensions()
+	dims = append(dims, f.getInstanceIdDimension())
```

Comment:
nit - we can change all of these to just 
```suggestion
	dims := append(f.getMetricSpecificDimensions(), f.getInstanceIdDimension())
```